src/Validator/InputCheck.js

```diff
@@ -0,0 +1,79 @@
+import { NUMBER, ERROR_MESSAGE, MENU_LIST } from '../Utils/constants.js';
+
+const checkInputFormat = (input, errorMessage) => {
+  if (input === undefined) {
+    throw new Error(`${ERROR_MESSAGE.errorText}${errorMessage}`);
+  }
+};
+
+const checkInputTypeNumber = (input, errorMessage) => {
+  if (!/^\d+$/.test(input)) {
+    throw new Error(`${ERROR_MESSAGE.errorText}${errorMessage}`);
+  }
+};
+
+const checkValidateOrderItem = (orderItem) => {
+  const [menu, quantity] = orderItem.split('-').map((item) => item.trim());
+  const isMenuValid = Object.values(MENU_LIST).flat().includes(menu);
+
+  if (!isMenuValid || Number.isNaN(Number(quantity)) || quantity < NUMBER.orderQuantity) {
+    throw new Error(`${ERROR_MESSAGE.errorText}${ERROR_MESSAGE.invalidOrder}`);
+  }
+
+  return { menu, quantity: Number(quantity) };
+};
+
+const checkForDuplicateMenus = (orderDetails) => {
+  const uniqueMenus = new Set(orderDetails.map((item) => item.menu));
+
+  if (uniqueMenus.size !== orderDetails.length) {
+    throw new Error(`${ERROR_MESSAGE.errorText}${ERROR_MESSAGE.invalidOrder}`);
+  }
+};
+
+const checkForOnlyDrink = (orderDetails) => {
+  const notDrink = new Set(
+    orderDetails.filter((item) => !MENU_LIST.음료.includes(item.menu)).map((item) => item.menu),
+  );
+
+  if ([...notDrink].length === 0) {
+    throw new Error(`${ERROR_MESSAGE.errorText}${ERROR_MESSAGE.invalidOrder}`);
+  }
```

Comment:
Set의 size로도 길이를 알 수 있을 것 같아요!

```suggestion
  if (notDrink.size === 0) {
    throw new Error(`${ERROR_MESSAGE.errorText}${ERROR_MESSAGE.invalidOrder}`);
  }
```